### PR TITLE
fix: do not throw err if func_config does not exist

### PR DESCRIPTION
**Pull request by minz1027**
It will throw an err during the `require('../../.func_config')` because we remove the `.func_config` file.
https://cd.screwdriver.cd/pipelines/1/builds/4770

### CODE REVIEW

File: features/support/before-hook.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 'use strict';
 
+const fs = require('fs');
 /* eslint-disable import/no-unresolved */
-const config = require('../../.func_config');
+const config = fs.existsSync('../../.func_config') ? require('../../.func_config') : {};
```
Comment (FenrirUnbound) [inline features/support/before-hook.js]: `fs.existsSync` is deprecated: https://nodejs.org/api/fs.html#fs_fs_exists_path_callback

**Comment by FenrirUnbound** [comment]
There are more elegant methods to handle this. [Node Env File](https://www.npmjs.com/package/node-env-file) handles secrets much better and the interface is always through environment variables

File: features/support/before-hook.js (hunk 1 above)
Comment (ian-fox) [inline features/support/before-hook.js]: It looks like `fs.exists` is deprecated, but not `fs.existsSync`.

Comment (ian-fox) [inline features/support/before-hook.js]: Though node env file does look cleaner you're right there.

**Comment by minz1027** [comment]
@FenrirUnbound Good idea. Switch to use `node-env-file`. Can you re-review it?

**Pull request merged**